pkg/cloudprovider/providers/azure/azure_vmss.go

```diff
@@ -185,7 +185,13 @@ func (ss *scaleSet) GetInstanceIDByNodeName(name string) (string, error) {
 		return "", err
 	}
 
-	return *vm.ID, nil
+	resourceID := *vm.ID
```

Comment:
```
return convertResourceGroupNameToLower(*vm.ID)
```

Reply:
add an error log here

Reply:
then add this log into function `convertResourceGroupNameToLower`?